packages/pn-commons/src/components/CodeModal/__test__/CodeInput.test.tsx

```diff
@@ -73,34 +73,34 @@ describe('CodeInput Component', () => {
       <CodeInput initialValues={new Array(5).fill('')} onChange={handleChangeMock} />
     );
     const codeInputs = getAllByTestId(/code-input-[0-4]/);
-    fireEvent.change(codeInputs[2], { target: { value: '3' } });
-    expect(codeInputs[2]).toHaveValue('3');
     await waitFor(() => {
```

Comment:
try to restore the previous version. i tried for this file and it worked. Do it for all tests